src/app/components/media/MediaComponent.js

```diff
@@ -49,16 +73,37 @@ class MediaComponent extends Component {
     const showRequests = (enabledBots.indexOf('smooch') > -1 || props.media.requests_count > 0);
     const showTab = showRequests ? 'requests' : 'tasks';
 
+    const { t = '', id } = qs.parse(document.location.hash.substring(1));
+    const [start, end] = t.split(',').map(s => parseFloat(s));
+
+    const gaps = [];
+    if (start) gaps.push([0, start]);
+    if (end) gaps.push([end, Number.MAX_VALUE]);
+
     this.state = {
-      showTab,
+      duration: 0,
+      playing: false,
+      progress: 0,
       showRequests,
+      showTab,
+      showVideoAnno: t && id,
+      fragment: { t, id },
+      start,
+      end,
+      gaps,
+      time: 0,
+      playerRef: null,
```

Comment:
See my comments on refs further down. TL;DR should not be state: should be `playerRef = React.createRef();`.

Reply:
Fixed in 5ce99d5fa2fba2c6110cbd4aa70b408a35125d1a